opensubtitles/search.go

```diff
@@ -0,0 +1,78 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+//OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#search
```

Comment:
Codacy found an issue: [comment on exported type SearchService should be of the form "SearchService ..." (with optional leading article)](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)